src/app/pages/PerpetualPage/components/AccountBalanceForm/index.tsx

```diff
@@ -38,30 +36,22 @@ export const AccountBalanceForm: React.FC<AccountBalanceFormProps> = ({
     collateral,
   ]);
 
-  const { checkMaintenance, States } = useMaintenance();
-  const fundAccountLocked =
-    checkMaintenance(States.PERPETUALS) ||
-    checkMaintenance(States.PERPETUALS_ACCOUNT_FUND);
-
-  const withdrawAccountLocked =
-    checkMaintenance(States.PERPETUALS) ||
-    checkMaintenance(States.PERPETUALS_ACCOUNT_WITHDRAW);
-
-  const transferAccountLocked =
-    checkMaintenance(States.PERPETUALS) ||
-    checkMaintenance(States.PERPETUALS_ACCOUNT_TRANSFER);
```

Comment:
Can you please implement the maintenance checks for the buttons. 
If there is a better maintenance key use that please.